Reason: Support prefixed ar
Upstream: I don't think there is an upstream

diff -ur a/configure.ac b/configure.ac
--- a/configure.ac	2004-02-23 09:57:29.000000000 -0800
+++ b/configure.ac	2013-10-27 23:22:40.237189516 -0700
@@ -17,6 +17,7 @@
 AC_PROG_MAKE_SET
 AC_PROG_RANLIB
 AM_PROG_CC_STDC
+AM_PROG_AR
 
 if test "x$am_cv_prog_cc_stdc" = xno ; then
   AC_ERROR([the C compiler doesn't handle ANSI-C])
